src/docnode.cpp

```diff
@@ -3214,58 +3210,59 @@ void DocPara::handleCite()
   parser()->tokenizer.setStatePara();
 }
 
-void DocPara::handleEmoji()
+void DocPara::handleEmoji(const QCString &cmdName, const int tokIn)
```

Comment:
The `const` is kind of redundant for value types, and I would prefer to resolve the `TK_COMMAND_CHAR(tokIn)` at the caller side and pass in a char, to prevent accidental mixing of `tokIn` and other `tok`'s.

Reply:
Indeed const is redundant, would you like to have here a `const char *` or a `char`?
Currently `TK_COMMAND_CHAR(tokIn)` returns a string.

(I already used `tokIn` to prevent this potential mixing.

Reply:
Maybe best to change `TK_COMMAND_CHAR` to return a char and then pass the char and use `%c` in the warnings.

Reply:
Rereading the comment I see a `char` is  preferred.